docs/basic_training/operators.pt.md

```diff
@@ -0,0 +1,271 @@
+---
+description: Material de treinamento básico do Nextflow
+---
+
+# Operadores
+
+Operadores são métodos para conectar canais, transformar valores emitidos por canais ou executar regras próprias em canais.
+
+Existem sete grupos de operadores descritos em detalhe na Documentação do Nextflow, estes são:
+
+1. [Operadores de filtragem](https://www.nextflow.io/docs/latest/operator.html#filtering-operators)
+2. [Operadores de transformação](https://www.nextflow.io/docs/latest/operator.html#transforming-operators)
+3. [Operadores de divisão](https://www.nextflow.io/docs/latest/operator.html#splitting-operators)
+4. [Operadores de combinação](https://www.nextflow.io/docs/latest/operator.html#combining-operators)
+5. [Operadores de bifurcação](https://www.nextflow.io/docs/latest/operator.html#forking-operators)
+6. [Operadores matemáticos](https://www.nextflow.io/docs/latest/operator.html#maths-operators)
+7. [Outros operadores](https://www.nextflow.io/docs/latest/operator.html#other-operators)
+
+## Exemplo básico
+
+!!! info ""
+
+    Clique no ícone :material-plus-circle: para ver explicações do código.
+
+```groovy linenums="1"
+nums = Channel.of(1,2,3,4) // (1)!
+quadrados = nums.map { it -> it * it } // (2)!
+quadrados.view() // (3)!
+```
+
+1. Cria um canal de fila que emite quatro valores
+2. Cria um novo canal, transformando cada número ao quadrado
+3. Imprime o conteúdo do canal
+
+<figure class="excalidraw">
+--8<-- "docs/basic_training/img/channel-map.excalidraw.svg"
+</figure>
+
+Para implementar funcionalidades específicas operadores também podem ser encadeados. Então, o código anterior também pode ser escrito assim:
+
+```groovy linenums="1"
+Channel
+    .of(1,2,3,4)
+    .map { it -> it * it }
+    .view()
+```
+
+## Operadores básicos
+
+Agora iremos explorar alguns dos operadores mais comuns.
+
+### `view()`
+
+O operador `view` imprime os itens emitidos por um canal para o terminal, acrescentando um caractere de _quebra de linha_ após cada item. Por exemplo:
+
+```groovy linenums="1"
+Channel
+    .of('foo', 'bar', 'baz')
+    .view()
+```
+
+```console title="Output"
+foo
+bar
+baz
+```
+
+Você também pode especificar uma _clausura_ para personalizar como os itens são impressos. Por exemplo:
+
+```groovy linenums="1"
+Channel
+    .of('foo', 'bar', 'baz')
+    .view { "- $it" }
+```
+
+```console title="Output"
+- foo
+- bar
+- baz
+```
+
+### `map()`
+
+O operador `map` aplica uma função de sua escolha em cada item emitido por um canal
+e retorna os items obtidos como um novo canal. A função aplicada é chamada de função
+de _mapeamento_ e é expressa com uma _clausura_, como demonstrado no exemplo abaixo:
+
+```groovy linenums="1"
+Channel
+    .of( 'olá', 'mundo' )
+    .map { it -> it.reverse() }
+    .view()
+```
+
+Um `map` pode associar uma _tupla_ genérica a cada elemento e pode conter qualquer
+tipo de dado.
+
+```groovy linenums="1"
+Channel
+    .of( 'olá', 'mundo' )
+    .map { word -> [word, word.size()] }
```

Comment:
```suggestion
    .map { palavra -> [palavra, palavra.size()] }
```

If the English words were translated in the other examples, I think we should do it everywhere.